frontend/package.json

```diff
@@ -40,6 +40,7 @@
     "react-select-async-paginate": "^0.7.2",
     "react-toastify": "^9.1.2",
     "typewriter-effect": "^2.19.0",
+    "use-image": "^1.1.0",
```

Comment:
Is this still needed? @absurd-turtle what do you think?

Reply:
It is sometimes a good idea to do a quick review of a PR yourself. Please ask for review when I should look again.